helm/aws-operator/templates/rbac.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
   labels:
     {{- include "aws-operator.labels" . | nindent 4 }}
 rules:
+  - apiGroups:
+      - release.giantswarm.io
+    resources:
+      - release
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
We _should_ only need get - let's try this first though.

Reply:
It works, do we want to restrict only to get?

Reply:
Yes, please. That should be enough.

Reply:
Checking now

Reply:
done